go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/nanmu42/limitio v1.0.0
 	github.com/orangecms/go-framebuffer v0.0.0-20200613202404-a0700d90c330
 	github.com/pborman/getopt/v2 v2.1.0
-	github.com/pierrec/lz4/v4 v4.1.14
+	github.com/pierrec/lz4/v4 v4.1.21
```

Comment:
can you revert the module updates? I know that 4.1.21 is def broken, as I filed a bug on .15 last year

Reply:
fortunately, this won't be happening, I'm going to redo this PR as a series of PRs, and I won't be messing up modules. Geez, lz4 is broken? OW OW OW!